GameReadyValidator/src/main/java/pl/przemyslawjakubowski/tajgertim/gamereadyvalidator/gamevalidation/GameReadyValidatorQueueListener.java

```diff
@@ -0,0 +1,27 @@
+package pl.przemyslawjakubowski.tajgertim.gamereadyvalidator.gamevalidation;
+
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.context.ApplicationEventPublisher;
+import pl.jakubowskiprzemyslaw.tajgertim.models.confirmation.Confirmation;
+import pl.jakubowskiprzemyslaw.tajgertim.services.BattleShipQueueInteractionHandler;
+import pl.jakubowskiprzemyslaw.tajgertim.services.LoggerService;
+
+@BattleShipQueueInteractionHandler
+class GameReadyValidatorQueueListener {
+
+    private final LoggerService logger;
+    private final ApplicationEventPublisher publisher;
+
+    GameReadyValidatorQueueListener(LoggerService logger, ApplicationEventPublisher publisher) {
+        this.logger = logger;
+        this.publisher = publisher;
+    }
+
+    @RabbitListener(queues = "GameReadyValidationQueue")
+    void getConfirmationsFromGameReadyValidatorQueue(Confirmation confirmation) {
+        logger.logInfo(GameReadyValidatorQueueListener.class, confirmation.toString());
+
+        ConfirmationReceivedEvent event = new ConfirmationReceivedEvent(this, confirmation);
```

Comment:
inline

Reply:
We've done it this way according to articles like this one:
https://blogs.msdn.microsoft.com/cellfish/2009/08/19/object-calisthenics-rule-4-use-only-one-dot-per-line/

Reply:
inlined